server/src/main/resources/routes

```diff
@@ -16,7 +16,7 @@ GET /users/me controllers.UsersController.getCurrentUser()
 PUT /users/me controllers.UsersController.update()
 
 # get the current user logs
-GET /users/me/logs controllers.UsersController.getLogs()
+GET /users/me/logs/scrollId/:scrollId/limit/:limit controllers.UsersController.getLogs(scrollId,limit)
```

Comment:
Must be `GET /users/me/logs controllers.UsersController.getLogs(scrollId: Option[String] ?= None, limit: Int ?= 10)`